client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -285,147 +255,36 @@ class AdsFormSettings extends Component {
 		);
 	}
 
-	siteOwnerOptions() {
-		const { translate } = this.props;
-
-		return (
-			<div>
-				<FormFieldset>
-					<FormLabel htmlFor="paypal">{ translate( 'PayPal E-mail Address' ) }</FormLabel>
-					<FormTextInput
-						name="paypal"
-						id="paypal"
-						value={ this.state.paypal || '' }
-						onChange={ this.handleChange }
-						disabled={ this.props.isLoading }
-					/>
-				</FormFieldset>
-				<FormFieldset>
-					<FormLabel htmlFor="who_owns">{ translate( 'Who owns this site?' ) }</FormLabel>
-					<FormSelect
-						name="who_owns"
```

Comment:
Should we keep this field? Seems like its absence is preventing me from saving the settings.

Reply:
Oops, I forgot to link to the WPCOM patch that will remove this requirement: D70725-code

Once that is applied these fields will no longer be required server-side.

Reply:
Works well with the patch! 🚀